src/main/java/com/commercetools/sync/commons/utils/SyncSolutionInfo.java

```diff
@@ -12,19 +11,15 @@
      *
      * <p>A User-Agent header with a solution information looks like this:
      * {@code commercetools-jvm-sdk/1.4.1 (AHC/2.0) Java/1.8.0_92-b14 (Mac OS X; x86_64)
-     * {@code implementationTitle}/{@code implementationVersion}
-     * (+https://github.com/commercetools/commercetools-sync-java; +ps-dev@commercetools.com)}</p>
+     * {implementationTitle}/{implementationVersion}}</p>
      *
      */
     public SyncSolutionInfo() {
-        final String libraryName = "commercetools-sync-java";
         final String implementationTitle = ofNullable(getClass().getPackage().getImplementationTitle())
-            .orElse(libraryName);
+            .orElse("commercetools-sync-java");
         final String implementationVersion = ofNullable(getClass().getPackage().getImplementationVersion())
             .orElse("DEBUG-VERSION");
         setName(implementationTitle);
         setVersion(implementationVersion);
-        setWebsite(format("https://github.com/commercetools/%s", libraryName));
```

Comment:
In the task, you wrote `remove email`, but you want to remove website too?

Reply:
because in the JVM SDK if the emergency contact is not set, the website will not be used as well: https://github.com/commercetools/commercetools-jvm-sdk/blob/93df1a10f6ed9b9a598915f3fc6e1d63c880d690/commercetools-java-client-core/src/main/java/io/sphere/sdk/client/UserAgentUtils.java#L51-L67